python/nav/ipdevpoll/snmp/common.py

```diff
@@ -16,13 +16,17 @@
 "common AgentProxy mixin"
 import time
 import logging
+from dataclasses import dataclass
 from functools import wraps
-from collections import namedtuple
+from typing import Optional, Any, Dict
 
 from twisted.internet import reactor
 from twisted.internet.defer import succeed
 from twisted.internet.task import deferLater
 
+from nav.Snmp.defines import SecurityLevel, AuthenticationProtocol, PrivacyProtocol
```

Comment:
Python 3.11+ have StrEnum, and it is possible to have a StrEnum in python 3.7 as well, see
`zinolib.compat.StrEnum`. Not that we need change anything now, but maybe when we get rid of python 3.7?

Reply:
We get to clean up a lot of things once we drop Python 3.7 support.  These imported Enums are already as close to StrEnums as they can be without adding a backport.